Plugins/NativeEngine/Source/VertexArray.cpp

```diff
@@ -85,7 +95,50 @@ namespace Babylon
 
     void VertexArray::SetVertexBuffers(bgfx::Encoder* encoder, uint32_t startVertex, uint32_t numVertices)
     {
-        for (auto& pair : m_vertexBufferRecords)
+        const bgfx::Caps* caps = bgfx::getCaps();
+
+        // Check if instancing is supported.
+        const bool instancingSupported = 0 != (BGFX_CAPS_INSTANCING & caps->supported);
```

Comment:
Doesn't Babylon.js already check this? Also, if we are dropping things, we should log a warning.

Reply:
warning logged. We don't expose many things from bgfx caps up to bjs. there is already an issue for that.